msi/release.sh

```diff
@@ -124,6 +124,10 @@ die "Could not generate ReleaseNotes.html."
 make -C ../ edit-git-bash.exe ||
 die "Could not build edit-git-bash.exe."
 
+# Compile proxy-lookup.exe
+make -C ../ proxy-lookup.exe ||
+die "Could not build proxy-lookup.exe."
```

Comment:
I think if you do build it in `msi/release.sh`, you probably also want to bundle it here.

An easier way might be to simply have the file installed into the correct location via the `git-extra` package, [just like `create-shortcut.exe`](https://github.com/git-for-windows/build-extra/blob/master/git-extra/PKGBUILD#L98).

Reply:
Could you do that? Preferably as a separate commit that needs to be reordered to be th first co|mit in this PR